packages/amplify-e2e-tests/src/__tests__/auth_6.test.ts

```diff
@@ -0,0 +1,64 @@
+import { $TSAny } from 'amplify-cli-core';
+import {
+  addAuthWithMaxOptions,
+  amplifyPushAuth,
+  createNewProjectDir,
+  deleteProject,
+  deleteProjectDir,
+  getProjectMeta,
+  initJSProjectWithProfile,
+} from 'amplify-e2e-core';
+
+const PROJECT_NAME = 'authTest';
+const defaultSettings = {
+  name: PROJECT_NAME,
+};
+describe('zero config auth ', () => {
+  let projRoot: string;
+  beforeEach(async () => {
+    projRoot = await createNewProjectDir('zero-config-auth');
+  });
+
+  afterEach(async () => {
+    await deleteProject(projRoot);
+    deleteProjectDir(projRoot);
+  });
+
+  it('...should init a javascript project and add auth with all options and update front end config', async () => {
+    await initJSProjectWithProfile(projRoot, defaultSettings);
+    await addAuthWithMaxOptions(projRoot, {});
+    await amplifyPushAuth(projRoot);
+
+    const meta = getProjectMeta(projRoot);
+    const authMeta: $TSAny = Object.values(meta.auth)[1];
+
+    expect(authMeta.frontendAuthConfig).toMatchInlineSnapshot(`
+      Object {
+        "loginMechanism": Array [
```

Comment:
Shouldn't this be plural `loginMechanisms`?

Reply:
@drochetti @manueliglesias Do you recall why this was singular vs. plural in the API Review?

This feature hasn't shipped, so I'm wondering if this is a tweak we need to land before releasing.

Reply:
cc @lazpavel